### PR TITLE
Komikku: update to 0.30.0.

### PR DESCRIPTION
**Pull request by not-chicken**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/Komikku/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+revision=1
 wrksrc=Komikku-v${version}
 build_style=meson
 hostmakedepends="gettext glib-devel gobject-introspection pkg-config"
 makedepends="gtk+3-devel libhandy1-devel"
 depends="gtk+3 libhandy1 libnotify libsecret python3-BeautifulSoup4
  python3-Pillow python3-Unidecode python3-cloudscraper python3-dateparser
  python3-gobject python3-keyring python3-lxml python3-magic
- python3-pure-protobuf"
+ python3-natsort python3-pure-protobuf"
```
Comment (paper42) [inline srcpkgs/Komikku/template:12]: python3-requests dependency is missing

File: srcpkgs/Komikku/template [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 checkdepends="appstream-glib desktop-file-utils"
 short_desc="Online/offline manga reader for GNOME"
 maintainer="Lorem <notloremipsum@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gitlab.com/valos/Komikku"
 distfiles="${homepage}/-/archive/v${version}/Komikku-v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/Komikku/template:18]: could you remove $homepage?

File: srcpkgs/Komikku/template (hunk 1 above)
Comment (not-chicken) [reply srcpkgs/Komikku/template:12]: It's pulled in via `python3-cloudscraper`

File: srcpkgs/Komikku/template (hunk 2 above)
Comment (not-chicken) [reply srcpkgs/Komikku/template]: done

**Comment by not-chicken** [comment]
An issue was spotted by @paper42, it's not a regression. Reported upstream: https://gitlab.com/valos/Komikku/-/issues/149

**Comment by ericonr** [comment]
@paper42 Ok to merge?

**Comment by paper42** [comment]
ok, let's merge this

@not-chicken could you watch the issue and submit a fix when we know what's happening?

**Comment by not-chicken** [comment]
> @not-chicken could you watch the issue and submit a fix when we know what's happening?

It was [fixed](https://gitlab.com/valos/Komikku/-/commit/35393c0af19b8e55d3be626f430a05a5969ad5dc) upstream! :rocket: 
Should I submit it as patch or wait for new release?

**Comment by paper42** [comment]
> Should I submit it as patch or wait for new release?

I think you can submit a PR with a patch.

**Comment by not-chicken** [comment]
#32471

**Pull request merged**